public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -75,10 +76,11 @@ export const LogRowMessage = React.memo((props: Props) => {
     onPinLine,
     pinned,
     mouseIsOver,
+    onBlur,
   } = props;
   const { hasAnsi, raw } = row;
   const restructuredEntry = useMemo(() => restructureLog(raw, prettifyLogMessage), [raw, prettifyLogMessage]);
-  const shouldShowMenu = useMemo(() => mouseIsOver || pinned, [mouseIsOver, pinned]);
+  const shouldShowMenu = useMemo(() => mouseIsOver || pinned || true, [mouseIsOver, pinned]);
```

Comment:
I suspect this is a leftover from testing?

Reply:
Oh damn, yes. Good catch!

Reply:
https://github.com/grafana/grafana/pull/72976

Reply:
Closed https://github.com/grafana/grafana/pull/72974 in favor of https://github.com/grafana/grafana/pull/72976  😆